main.css

```diff
@@ -45,10 +46,25 @@ a{
     
 }
 .link-btn:hover{
-    color: black;
+    color: var(--light-rose);
+    font-weight: 700;
+    border: 4px solid var(--light-rose);
+    background: transparent;
 }
 .btn-wishlist
 {
     cursor: pointer;
 }
+.card-image
+{
+    object-fit: fill;
```

Comment:
I will suggest using cover for the object-fit property, apply once, and see the changes yourself.